### DISCUSSION TITLE
Welcoming Tobias Bieniek as a reviewer / maintainer

### DISCUSSION THREAD

**Discussion by jplatte** [tokio-rs/axum#3099] [Announcements]
Small announcement: I invited @Turbo87 as a reviewer / maintainer yesterday.

He started commenting, reviewing and sending PRs all within a short timeframe recently, and has a great open source track record as well. I feel like we've all been stretched a bit thin so it's great to have another person on the team.

This brings the total number of maintainers up to 5. In order of "appearance":

- David Pedersen (original author, currently inactive)
- Me (the only other person with direct ownership of the crates on crates.io, though we have the tokio maintainers team as backup as well)
- @mladedav (David Mládek)
- @yanns (Yann Simon)
- @Turbo87 (Tobias Bieniek)

**Comment by Turbo87** [comment]
👋

**Comment by ghost** [comment]
Been using Axum since BoxRouter, it's simply the best routing library out there. 

Thanks for making web development in Rust a pleasure!

**Comment by dayvejones** [comment]
Great team. Excellent people. Thank you!
